src/managed/CommonManaged.props

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>$(VersionPrefix)</VersionPrefix>
```

Comment:
This doesn't work.  It is setting a property to itself, which is not yet defined.  Just hard-code this to `2.1.0`.  We have a bug to fix this for real, which will cause more work:  https://github.com/dotnet/core-setup/issues/2435